File: revisao/revisao01/ConviteDaOBI.java
import java.util.Scanner;

class ConviteDaOBI{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);

        System.out.println("Informe o número de competidores: ");
        int numeroCompetirodes = scan.nextInt();
        System.out.println("Informe a pontuação mínima: ");
        int pontuacaoMinima = scan.nextInt();

        int numeroDeAprovados =0;

        for(int count =0; count < numeroCompetirodes; count++){
            System.out.printf("Informe a nota 1 do competidor [%d]: \n",count+1);
            int primeiraNotaDoCompetidor = scan.nextInt();
            System.out.printf("Informe a nota 2 do competidor [%d]: \n",count+1);
            int segundaNotaDoCompetidor = scan.nextInt();

            int notaTotalDoCompetidor = primeiraNotaDoCompetidor +segundaNotaDoCompetidor;

            System.out.printf("Verificando se o competidor [%d] foi aprovado \n\n",count+1);
            if(notaTotalDoCompetidor >= pontuacaoMinima){
                numeroDeAprovados++;
            }
        }
        System.out.println("Número de competidores aprovados: "+numeroDeAprovados);

    }
}

File: revisao/revisao01/ConviteDaOBI.py
numero_competidores = int(input('Informe o número de competidores: '))
pontuacao_minima = int(input('Informe a pontuação mínima: '))

quantidade_aprovados = 0

for numero in range(0, numero_competidores):
    primeira_nota_competidor = int(input(f'\nInforme a primeira nota do competidor [{numero}]: '))
    segunda_nota_competidor = int(input(f'Informe a segunda nota do competidor [{numero}]: '))

    nota_total_competidor = primeira_nota_competidor + segunda_nota_competidor
    print('Verificando se competidor foi aprovado.. \n')
    if(nota_total_competidor >=pontuacao_minima):
        quantidade_aprovados+= 1
    

print('Número de competidores aprovados: '+str(quantidade_aprovados))
